mender-update/daemon/states.hpp

```diff
@@ -280,7 +280,8 @@ class StateScriptState : virtual public StateType {
 		script_executor::Action action,
 		chrono::seconds retry_interval,
 		const string &artifact_script_path,
-		const string &rootfs_script_path) :
+		const string &rootfs_script_path,
+		script_executor::OnError on_error_ = script_executor::OnError::Fail) :
```

Comment:
Why give it a default value? I would vote for it to be mandatory.

Reply:
+1

Reply:
:ok: